spring-vault-core/src/main/java/org/springframework/vault/core/VaultKeyValue2Template.java

```diff
@@ -89,4 +90,29 @@ public void put(String path, Object body) {
 		doWrite(createDataPath(path), Collections.singletonMap("data", body));
 	}
 
+	/**
+	 * Performs a KV Patch operation.
+	 * @param path must not be {@literal null} or empty.
+	 * @param kv New key value map to be updated
+	 * @since 2.3
+	 */
+	public void patch(String path, Map<String, ?> kv) {
```

Comment:
This operation should be also provided for the version 1 backend. Since this class is package-protected, there's no way to consume this method. Therefore, the method should be also made available in the `VaultKeyValueOperations` interface.

Reply:
Applied:
1. VaultKeyValueOperations.java : https://github.com/spring-projects/spring-vault/pull/587/commits/95d25bd9120c340cecc2eab71c67ffc047e41f07#diff-93f96dc2bd921937c3f5ee0eda355c0dR70
1. VaultKeyValue2Template.java : https://github.com/spring-projects/spring-vault/pull/587/commits/95d25bd9120c340cecc2eab71c67ffc047e41f07#diff-917f198074796659086e92ad34820ddfR101
1. VaultKeyValue1Template.java : https://github.com/spring-projects/spring-vault/pull/587/commits/95d25bd9120c340cecc2eab71c67ffc047e41f07#diff-36bdc62ca9e4f9e53d696f8675e326eaR111